config/settings.yml

```diff
@@ -54,6 +54,7 @@ defaults: &defaults
     - moderator
   disallowed_hashtags: # space separated string or list of hashtags without the hash
   bootstrap_timeline_accounts: ''
+  keybase_contacts: 'admin@localhostmastodon, yournamehere@keybase'
```

Comment:
Should probably be an empty string. This file includes valid default settings, not examples.